core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/service/impl/DSWDatasourceServiceImpl.java

```diff
@@ -89,6 +89,8 @@
 
 import com.thoughtworks.xstream.XStream;
 
+import static java.util.stream.Collectors.toList;
```

Comment:
Static imports are anti-style for non-test cases.

Reply:
See: https://github.com/pentaho/data-access/pull/1021